packages/react-icons/src/utils/constants.tsx

```diff
@@ -0,0 +1,2 @@
+export const FILLED_CLASSNAME = "fui-Icon-filled";
```

Comment:
is there a particular reason this was changed from fluentui-icon-xxx to fui-Icon-xxx?

Reply:
and are there any particular knock-ons to worry about? Presumably the classnames would have been already used/referenced, so is this not breaking?

Reply:
this was changed to be inline with other fluentui component classnames naming convention. This is not breaking since this will get converted anyway by the babel-plugin